### DISCUSSION TITLE
Is there any way to use unsupervised learning to improve image classification accuracy

### DISCUSSION THREAD

**Discussion by Enderfga** [autogluon/autogluon#2082] [Q&A]
I am working on an image classification task with 30G of unlabeled data and 7G of training set, I want to achieve the highest AUC on 10,000 test sets, currently the result achieved using 'best_quality' is 0.812335, I would like to ask if there is any way to improve the AUC using the large amount of unlabeled data
